### ISSUE TITLE
a11y: Missing textual descriptions for image button elements

### ISSUE THREAD

**Issue by nwmac** [rancher/dashboard#12806]
a11y Issue: 84

The image buttons listed below lack textual description resulting in screen reader users finding it difficult to identify the purpose of these image buttons.

Account and API Keys
 - API keys
  - More Actions image button

Projects/Namespaces
- Flat List, Group By Project and  More Actions image buttons

Health Scale Component
Main Content 
- Increase and Decrease image buttons (which gets available after expanding the healthscale dropdown)

Charts
Main Content
- Refresh image button 

Extensions
Global Settings
- More Actions image button
 
**Example:**

![image](https://github.com/user-attachments/assets/5efe411e-a4e0-4f38-9160-f5ab2e94f889)

**Remediation:**

Ensure that appropriate textual description such as "More Actions - kubeconfig-user-2rgrpmfj4q", "More Actions - kubeconfig-user-2rgrp55dcd", "More Actions - kubeconfig-user-2rgrp7f6k6" and so on is provided for the image buttons using "aria-label" attribute.

**Comment by izaac** [comment]
Chrome version: `133.0.6943.127`
Firefox version: `135.0.1`
Rancher version: `v2.11-3fd614ce7cab69d448bf39570fbe8b657112d710-head`
UI version: `master 470c849`

Using a screen reader extension I validated the text is mentioned when selecting the items described in the issue. ✅
